DiscordChatExporter.Core/Exporting/Writers/JsonMessageWriter.cs

```diff
@@ -230,6 +230,7 @@ public override async ValueTask WriteMessageAsync(Message message)
             _writer.WriteString("name", message.Author.Name);
             _writer.WriteString("discriminator", message.Author.DiscriminatorFormatted);
             _writer.WriteString("nickname", Context.TryGetMember(message.Author.Id)?.Nick ?? message.Author.Name);
+            _writer.WriteString("color", Context.TryGetUserColor(message.Author.Id)?.ToHex() ?? "#FFFFFF");
```

Comment:
Let's not fall back to `"#FFFFFF"`  because that assumes we know what Discord would use a fallback color. And that's likely different depending on the theme.

Instead just write null if the color is null.
```suggestion
            _writer.WriteString("color", Context.TryGetUserColor(message.Author.Id)?.ToHex());
```